cmd/troubleshoot/cli/run.go

```diff
@@ -147,71 +137,50 @@ func runTroubleshoot(v *viper.Viper, arg string) error {
 		}
 	}()
 
-	archivePath, err := runCollectors(v, supportBundleSpec.Spec.Collectors, additionalRedactors, progressChan)
+	collectorCB := func(c chan interface{}, msg string) {
+		c <- fmt.Sprintf("%s", msg)
+	}
+
+	createOpts := supportbundle.SupportBundleCreateOpts{
+		CollectorProgressCallback: collectorCB,
+		CollectWithoutPermissions: v.GetBool("collect-without-permissions"),
+		KubernetesRestConfig:      k8sConfig,
+		Namespace:                 v.GetString("namespace"),
+		ProgressChan:              progressChan,
+		SinceTime:                 sinceTime,
+	}
+
+	archivePath, err := supportbundle.CollectSupportBundleFromSpec(&supportBundle.Spec, additionalRedactors, createOpts)
 	if err != nil {
 		return errors.Wrap(err, "run collectors")
 	}
 
 	fmt.Printf("\r%s\r", cursor.ClearEntireLine())
```

Comment:
should this print to `c` instead of using the fmt package directly?

Reply:
Not sure - this was existing. Proposing this instead?
```golang
c := color.New(color.FgHiWhite)
c.Println("\r%s\r", cursor.ClearEntireLine())
```

Reply:
yeah, seems like an oversight from before.  not related to your changes.